### DISCUSSION TITLE
Levenberg Marquardt Minimizer with complex numbers

### DISCUSSION THREAD

**Discussion by Hartwich** [mathnet/mathnet-numerics#1062] [Q&A]
Hey, is it possible to use the Levenberg Marquadt Minimizer with complex numbers? In the documentation it says that a `Vector<T>` can be used but when I use a `Vector<Complex>` I get an error that no conversion to `Vector<double>` is possible. So it seems like it is only implemented for `double` or am I wrong? Or does anyone have a way to use the algorithm with complex numbers?

Regards and thanks in advance.

**Comment by diluculo** [comment]
When dealing with complex data, you can convert it into `a sequential array of real and imaginary parts` or represent it in terms of `magnitude and phase`.
